sync_telegram/data/sync_project_data.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2020 Ivan Yelizariev <https://twitter.com/yelizariev>
+     License MIT (https://opensource.org/licenses/MIT). -->
+<odoo>
+    <record id="telegram_project" model="sync.project">
+        <field name="name">Telegram Integration module</field>
+        <field name="active" eval="False" />
+        <field name="eval_context">telegram</field>
+        <field name="common_code">
+<![CDATA[
+def user2name(user):
+    if user.username:
+        return '@%s' % (user.username)
+
+    name = user.first_name
+    if user.last_name:
+        name += ' %s' % user.last_name
+
+    return name
+
+def html_sanitize_telegram(html):
+    allowed_tags = set({"b", "i", "u", "s", "a", "code", "pre"})
+    cleaner = Cleaner(safe_attrs_only=True, safe_attrs=set(), allow_tags=allowed_tags, remove_unknown_tags=False)
+    html = cleaner.clean_html(html)
+    # remove surrounding div
+    return html[5:-6]
+]]>
+        </field>
+    </record>
+    <record id="telegram_project_param_welcome" model="sync.project.param">
+        <field name="key">TELEGRAM_WELCOME_MESSAGE</field>
+        <field name="value">Hello! How can I help you?</field>
+        <field name="description">
+            Message that is sent to a user on first bot usage
+        </field>
+        <field name="project_id" ref="sync.telegram_project" />
```

Comment:
`ValueError: External ID not found in the system: sync.telegram_project`
I see this error while installing this module.
You need to use references inside the sync_telegram module to get rid of this errors.
Should be something like that: `ref="sync_telegram.telegram_project"`

Reply:
Done. thx